### PR TITLE
Changed font-weight to 400 to support light weight font

### PR DESCRIPTION
**Pull request by goelesha**
Fixes: https://github.com/vector-im/element-web/issues/21171
Signed off By: Esha Goel (goelesha2001@gmail.com)

Before:
Screenshot 2022-04-16 at 12 39 33 PM

The issue is fixed on Safari browser as shown below:
image

Although issue still persists in chrome. Not sure how to rectify it on chrome
image

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Changed font-weight to 400 to support light weight font ([\#8345](https://github.com/matrix-org/matrix-react-sdk/pull/8345)). Fixes vector-im/element-web#21171. Contributed by @goelesha.

### CODE REVIEW

File: res/css/structures/_CreateRoom.scss [hunk 1]
```diff
@@ -25,7 +25,7 @@ limitations under the License.
 .mx_CreateRoom textarea {
     border-radius: 3px;
     border: 1px solid $strong-input-border-color;
-    font-weight: 300;
+    font-weight: 400;
```
Comment (robintown) [inline res/css/structures/_CreateRoom.scss:28]: As per https://github.com/matrix-org/matrix-react-sdk/pull/7880#issuecomment-1049808853, this entire file is unused, and so it should be deleted.

File: res/css/_common.scss [hunk 2]
```diff
@@ -294,7 +294,7 @@ legend {
     background-color: $background;
     color: $light-fg-color;
     z-index: 4012;
-    font-weight: 300;
+    font-weight: 400;
```
[the remark this answers is not in the document]
Comment (goelesha) [reply res/css/_common.scss]: > The default inherited font-weight is 400, so we can just remove this line entirely.

I have removed the entire line. Request you to kindly have a look at it now.
Thanks for your review

File: res/css/structures/_CreateRoom.scss (hunk 1 above)
Comment (goelesha) [reply res/css/structures/_CreateRoom.scss]: > As per [#7880 (comment)](https://github.com/matrix-org/matrix-react-sdk/pull/7880#issuecomment-1049808853), this entire file is unused, and so it should be deleted.

I have removed the file as a part of this PR. Kindly have a look. 
Thanks for your review.

**Review by robintown — APPROVED** [review]
Thank you!

**Comment by robintown** [comment]
You have failing tests, please make sure to update `_components.scss` to indicate that `_CreateRoom.scss` was removed

**Comment by goelesha** [comment]
> You have failing tests, please make sure to update `_components.scss` to indicate that `_CreateRoom.scss` was removed

Sure. I will do that

**Comment by goelesha** [comment]
> You have failing tests, please make sure to update `_components.scss` to indicate that `_CreateRoom.scss` was removed

Done the rqd changes. All the tests are passed now. Kindly have a look.

**Pull request merged**